eclog/src/test/java/com/ericsson/bss/cassandra/ecaudit/eclog/TestLogPrinter.java

```diff
@@ -163,30 +161,31 @@ private QueueReader givenRecords(boolean withBatchId, int count1, int count2) th
         }
         hasNextStub.thenReturn(false);
 
-        List<AuditRecord> records = new ArrayList<>();
+        List<StoredAuditRecord> records = new ArrayList<>();
         for (int i = 0; i < count1 + count2; i++)
         {
-            records.add(mockRecord(i, "1.2.3.4", 222, "5.6.7.8", "king", Status.ATTEMPT, withBatchId ? UUID.fromString("b23534c7-93af-497f-b00c-1edaaa335caa") : null, "SELECT QUERY"));
+            records.add(mockRecord((long) i, "1.2.3.4", 222, "5.6.7.8", "king", Status.ATTEMPT, withBatchId ? UUID.fromString("b23534c7-93af-497f-b00c-1edaaa335caa") : null, "SELECT QUERY"));
         }
-        OngoingStubbing<AuditRecord> recordStub = when(reader.nextRecord());
-        for (AuditRecord record : records)
+        OngoingStubbing<StoredAuditRecord> recordStub = when(reader.nextRecord());
+        for (StoredAuditRecord record : records)
         {
             recordStub = recordStub.thenReturn(record);
         }
 
         return reader;
     }
 
-    private AuditRecord mockRecord(long timestamp, String clientHost, int clientPort, String coordinatorHost, String user, Status status, UUID batchId, String operation) throws UnknownHostException
+    private StoredAuditRecord mockRecord(Long timestamp, String clientHost, Integer clientPort, String coordinatorHost, String user, Status status, UUID batchId, String operation) throws UnknownHostException
     {
-        AuditRecord record = mock(AuditRecord.class);
-        when(record.getTimestamp()).thenReturn(timestamp);
-        when(record.getClientAddress()).thenReturn(new InetSocketAddress(InetAddress.getByName(clientHost), clientPort));
-        when(record.getCoordinatorAddress()).thenReturn(InetAddress.getByName(coordinatorHost));
-        when(record.getUser()).thenReturn(user);
+        StoredAuditRecord record = mock(StoredAuditRecord.class);
+        when(record.getTimestamp()).thenReturn(Optional.ofNullable(timestamp));
+        when(record.getClientAddress()).thenReturn(Optional.ofNullable(InetAddress.getByName(clientHost)));
+        when(record.getClientPort()).thenReturn(Optional.ofNullable(clientPort));
+        when(record.getCoordinatorAddress()).thenReturn(Optional.ofNullable(InetAddress.getByName(coordinatorHost)));
+        when(record.getUser()).thenReturn(Optional.ofNullable(user));
         when(record.getBatchId()).thenReturn(Optional.ofNullable(batchId));
-        when(record.getStatus()).thenReturn(status);
-        when(record.getOperation()).thenReturn(new SimpleAuditOperation(operation));
+        when(record.getStatus()).thenReturn(Optional.ofNullable(status));
+        when(record.getOperation()).thenReturn(Optional.ofNullable(operation));
         return record;
```

Comment:
The mock already returns that (the default behavior for a method returning an Optional is Optional.empty()), but maybe we should be more specific here?
But again, this will be refactored with #102, so I think I'll leave it for now...